Pages/List/list.css

```diff
@@ -0,0 +1,50 @@
+.list-container {
+    display: flex;
+    padding: 2rem;
+    border: 0.2rem solid var(--blue-dominant);
+    border-radius: 2rem;
+    gap: 5rem;
+    justify-content: center;
+    margin: 2rem 3rem;
+}
```

Comment:
I think you can reduce/remove left margin so that list-container aling with other headings and paragraph in main section.